list2md.go

```diff
@@ -34,9 +34,14 @@ Please update **list.txt** (via Pull Request)
 | ------------ | ----- | ----- | ----------- | ----------- |
 `
 	tail = "\n*Last Automatic Update: %v*"
+
+	WARNING = "⚠️ No longer maintained ⚠️\n"
```

Comment:
I removed the `\n` because the markdown table does not add new line on a cell, but add a new row.